bentoml/repository/__init__.py

```diff
@@ -149,7 +150,12 @@ def add(self, bento_name, bento_version):
             raise BentoMLRepositoryException(
                 "Not able to get pre-signed URL on S3. Error: {}".format(e)
             )
-        return response
+
+        return BentoUri(
+            type=self.uri_type,
+            uri=response['url'],
+            additional_fields=json.dumps(response['fields']),
```

Comment:
could you help me understand the additional_fields here?

Reply:
Yes, the additionla_fields contains some extra information including the resigned signature. It's necessary for uploading a file. https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-presigned-urls.html